test/apollo/connector.go

```diff
@@ -4,41 +4,24 @@ import (
 	"context"
 	"net/http"
 
-	"github.com/amp-labs/connectors/common/scanning"
 	"github.com/amp-labs/connectors/common/scanning/credscanning"
 	"github.com/amp-labs/connectors/providers"
 	"github.com/amp-labs/connectors/providers/apollo"
 	testUtils "github.com/amp-labs/connectors/test/utils"
-	"github.com/amp-labs/connectors/utils"
 )
 
-func GetApolloConnector(ctx context.Context, filePath string) *apollo.Connector {
```

Comment:
How would one override this file path if they needed to?

Reply:
Every test file had code as follows:
```
filePath := os.Getenv("MS_CRM_CRED_FILE")
if filePath == "" {
	filePath = "./ms-crm-creds.json"
}
```
These things are preserved. So no changing path via code but via arguments.
Environment variable allows such configuration. With apollo it would be `APOLLO_CRED_FILE`.
See:
https://github.com/amp-labs/connectors/blob/main/common/scanning/credscanning/path.go